atlp-rwanda/rca-barefoot-team6-fe
# Ft resetpassword

#### What does this PR do?
 Reset password 
 ...

#### Description of  the task to be completed?
I have created the reset password forms 
...

#### How should this be manually tested?
by clicking on forgot password on the login page and then use the email to get a link on your email. Check your email, click on the link and create a new password. Finally, login with the new password
...

#### Any background context you want to provide?

...

#### What are the relevant pivotal tracker stories?
#184986732
...

#### Screenshots (if appropriate)

...
